### PR TITLE
Update Documentation

### PR DESCRIPTION
**Pull request by tonerdo**
Updating documentation ahead of `coverlet.collector` NuGet package release

### CODE REVIEW

**Comment by MarcoRossignoli** [comment]
Great improve I like it al lot...maybe is wip at the moment on readme I don't see build badge should we add?

File: Documentation/VSTestIntegration.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 # Coverlet Integration with VSTest
```
Comment (MarcoRossignoli) [inline Documentation/VSTestIntegration.md]: I think we should specify minimum SDK supported version https://github.com/tonerdo/coverlet/issues/395#issuecomment-494666202 2.2.300, right @vagisha-nidhi?

Comment (MarcoRossignoli) [reply Documentation/VSTestIntegration.md]: And maybe we'll add "simple" documentation like tool and msbuild for users, this is a design doc.

Comment (vagisha-nidhi) [reply Documentation/VSTestIntegration.md]: Yes. minimum SDK supported version should be specified.

**Comment by vagisha-nidhi** [comment]
@tonerdo @MarcoRossignoli May I know the tentative date for beta release?

**Comment by MarcoRossignoli** [comment]
cc: @tonerdo

**Comment by tonerdo** [comment]
@vagisha-nidhi sorry, I've been a bit preoccupied with a few things. I plan on releasing a `1.0` non beta today/tomorrow

**Pull request merged**